test/querybuilder.test.js

```diff
@@ -291,6 +291,15 @@ describe("Query Builder", () => {
 						upsertOnlyUpdateData: { id: 1 },
 					})
 				).to.not.throw();
+
+				expect(() =>
+					GenerateQuery(QueryType.UPSERT, "test", {
```

Comment:
Not a fault of your test in particular, but we really should be testing the output of these queries too